### PR TITLE
chore: Update dependencies [ignore-size]

### PR DESCRIPTION
**Pull request by ookami-kb**
#### Summary

Updated optimus and storybook to the latest versions. Most changes are pretty straightforward.

#### Testing steps

There were breaking changes in slidable library, check behavior of slidable component.

#### Follow-up issues

No.

#### Check during review

- Verify against YouTrack issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).

### CODE REVIEW

File: storybook/macos/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chaeMil) [inline storybook/macos/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme:1]: Why are we including this file?

Comment (ookami-kb) [reply storybook/macos/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme]: Yeah, I guess we can remove them, we're not specifically targeting macos.

**Review by chaeMil — APPROVED** [review]

**Comment by BegovicAmar** [comment]
@ookami-kb has this been merged already?
Im trying to go through and move all old issues to tested so we can clean up our board

**Comment by ookami-kb** [comment]
Yeah, should be.

**Pull request merged**